### PR TITLE
ci: do not run 3.6 tests by default

### PR DESCRIPTION
**Pull request by busunkim96**
Python 3.6 went End-of-Life in December. This PR adds 3.6 to the default `ignored_versions` list in `noxfile-template.py`, `noxfile.py` and`noxfile_config.py` files.

If we need to run 3.6 in any directories, we can use a noxfile_config that _doesn't_ have 3.6 in the ignore list.

### CODE REVIEW

**Review by anguillanneuf — APPROVED** [review]
Looks good for Pub/Sub.

**Comment by leahecole** [comment]
LGTM for data science onramp

**Review by parthea — APPROVED** [review]

**Review by shollyman — APPROVED** [review]

File: dataflow/encryption-keys/noxfile_config.py
Comment (busunkim96) [inline dataflow/encryption-keys/noxfile_config.py:25]: Hi @davidcavazos, is it alright to stop running these tests on 3.6?

File: ml_engine/online_prediction/noxfile_config.py
Comment (busunkim96) [inline ml_engine/online_prediction/noxfile_config.py:25]: @alecglassford We have you listed in CODEOWNERS for `ml_engine`, would you know if this change is OK to take?

File: notebooks/noxfile_config.py
Comment (busunkim96) [inline notebooks/noxfile_config.py:26]: This directory doesn't seem to have any `.py` files, maybe the `noxfile.py` and `noxfile_config.py` should be deleted? https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/notebooks

Comment (leahecole) [reply notebooks/noxfile_config.py:26]: Let's check in with @ivanmkc and/or @loferris who know a ton about the current notebooks testing strategy. My instinct is to say that's probably fine to remove those but I want an LGTM from one of those two folks 😄

File: ml_engine/online_prediction/noxfile_config.py
Comment (alecglassford) [reply ml_engine/online_prediction/noxfile_config.py:25]: @busunkim96 Hi! I no longer work at the company, so I'm not a good person to answer this question. (You should probably also remove me from `CODEOWNERS`; sorry that I didn't think to do this before I left!)

File: dataflow/encryption-keys/noxfile_config.py
Comment (davidcavazos) [reply dataflow/encryption-keys/noxfile_config.py:25]: Yes, that's okay. Thank you!

File: ml_engine/online_prediction/noxfile_config.py
Comment (leahecole) [reply ml_engine/online_prediction/noxfile_config.py:25]: @nicain can you take a look at this?

File: dataflow/encryption-keys/noxfile_config.py
Comment (nicain) [reply dataflow/encryption-keys/noxfile_config.py:25]: @busunkim96 This looks unresolved; I am going to mark my review as Request Changes because of this, but if I am misunderstanding something let me know.

File: ml_engine/online_prediction/noxfile_config.py
Comment (nicain) [reply ml_engine/online_prediction/noxfile_config.py:25]: TODO for @nicain find new codeowner for ml_engine, or remove

**Review by nicain — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (dandhlee) [inline .github/CODEOWNERS:58]: @nicain could you take this change onto a separate PR?

**Review by davidcavazos — APPROVED** [review]

**Comment by davidcavazos** [comment]
LGTM for Dataflow related samples.

File: dataflow/encryption-keys/noxfile_config.py
Comment (busunkim96) [reply dataflow/encryption-keys/noxfile_config.py:25]: Hi @nicain, this is resolved. The list of python versions are versions that are skipped in tests. I added 3.6 here after checking with David.

File: notebooks/noxfile_config.py
Comment (leahecole) [reply notebooks/noxfile_config.py:26]: I'm adding a separate issue to remove so it doesn't block this PR https://github.com/GoogleCloudPlatform/python-docs-samples/issues/7507

Comment (leahecole) [reply notebooks/noxfile_config.py:26]: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/7508

**Comment by leahecole** [comment]
@engelke does this look ok for appengine?

**Comment by busunkim96** [comment]
@nicain Could you re-review this PR?

**Review by nicain — APPROVED** [review]

**Pull request merged**
